### PR TITLE
Updating errors components & layout tweak to accomodate

### PR DESCRIPTION
**Pull request by ashleygyngell**

### CODE REVIEW

**Comment by ashleygyngell** [comment]
Updating errors component here as needed in other PR's too so will reduce their overall size. Thanks!

**Review by psoleckimoj — DISMISSED** [review]
LGTM!

File: app/assets/sass/application.scss [hunk 1]
```diff
@@ -285,6 +285,10 @@ span.bold {
     background:#f40b0c;
 }
 
+.vertical-align-middle {
```
[the remark this answers is not in the document]
Comment (ashleygyngell) [reply app/assets/sass/application.scss:288]: Nice spot, yes it does! Will update :)

[the remark this answers is not in the document]
Comment (psoleckimoj) [reply app/assets/sass/application.scss:288]: ..and then remove this 🙂

[the remark this answers is not in the document]
Comment (ashleygyngell) [reply app/assets/sass/application.scss:288]: Ah @psoleckimoj  - it does need to be applied on its own in other places on that page too! And then used in another table in other PR's, so will need to retain.

**Review by psoleckimoj — APPROVED** [review]
LGTM!

**Pull request merged**